masterlist.yaml

```
@@ -15906,24 +15924,26 @@ plugins:
     clean:
       - crc: 0xB4C1E773
         util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkTitleRemover.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+  - name: 'ClockworkSuperiorSorting.esp'
+    req:
+      - *PapyrusExtender
+      - *SKSE
     clean:
-      - crc: 0x913191E9
-        util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkMausoleumDummyDoorFix.esp'
+      - crc: 0x9E78E8D9
+        util: 'SSEEdit v4.0.4'
+  - name: 'ClockworkTitleRemover.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
-    clean:
-      - crc: 0xCAE2B6AE
-        util: 'SSEEdit v4.0.3g'
+    dirty:
+      - <<: *quickClean
+        crc: 0xBF1FBE1F
+        udr: 3
 
   - name: 'Darkend.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/10423/' ]
     dirty:
       # version: 1.4
       - <<: *quickClean
         crc: 0x616730B8
-        util: '[SSEEdit v4.0.0](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 26
```

This line was unintentionally deleted.

```suggestion
        util: '[SSEEdit v4.0.0](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 26
```

---

Aaah. Must've hit cut instead of copy.